src/sentry/static/sentry/app/components/sidebar/sidebarItem.jsx

```diff
@@ -175,6 +175,17 @@ const StyledSidebarItem = styled(({active, ...props}) => <Link {...props} />)`
     color: ${p => p.theme.gray1};
   }
 
+  &.focus-visible {
+    outline: none;
+    background: #584c66;
```

Comment:
Do we have a theme colour for this?

Reply:
no. I could be nice to get some color mutators in so we could lighten / darken colors, add opacity, etc. I'm doing opacity mutation in a hacky way in here